packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -231,6 +234,9 @@ export class QueryFormInputs extends React.Component<QueryFormInputsProps, State
                             const multiple = col.isJunctionLookup();
                             const joinValues = multiple;
                             const id = col.fieldKey + i + (componentKey ?? '');
+                            const queryFilter = col.lookup.hasQueryFilters()
+                                ? List(col.lookup.getQueryFilters())
+                                : queryFilters?.[col.fieldKey];
```

Comment:
The supplied queryFilters are no intended to added to the silly "remove all the samples without types" filter?

Reply:
Since queryFilters are new and used only for the add to storage scenario (filter Units rather than sample), it's fine to not add to, but use in absence of, for now. If there are new scenarios that will need both of those, changes can be made then.